src/_test_/channelActions.test.js

```diff
@@ -0,0 +1,12 @@
+import * as channelActions from "../redux/messageActions";
+import { setIsChannelsLoading, channel } from "../redux/channelActions";
```

Comment:
are these imports used? The only one I see being used is `setIsChannelsLoading`...

Reply:
Yes, I didn't finish writing these. This probably shouldn't have been in this PR. 😅